be/src/storage/persistent_index.cpp

```diff
@@ -2382,6 +2396,47 @@ Status PersistentIndex::_load(const PersistentIndexMetaPB& index_meta) {
         _l1_merged_num.emplace_back(-1);
         _has_l1 = true;
     }
+    // if reload, don't update _usage_and_size_by_key_size
```

Comment:
It's better to use size and nums

Reply:
> It's better to use size and nums

I check the code and there are too many places in the code where `usage` is used.  I will not change it in this pr to avoid adding burden of this pr review and i will refactor them in a new pr in the future。